roles/public_facing/tasks/main.yml

```diff
@@ -0,0 +1,13 @@
+---
+# ansible_host changes to 'localhost' in the next task.  Set target_host
```

Comment:
When you run something with `local_action:`, the `ansible_host` var temporarily changes to 'localhost'